packages/create-novu-app/package.json

```
@@ -0,0 +1,56 @@
+{
+  "name": "create-novu-app",
+  "version": "0.0.1-alpha.4",
```

We should probably agree on what the correct value is to put into the version

---

Let's go with the same version as other monorepo packages for now.

```suggestion
  "version": "0.24.2",
```

---

done